### ISSUE TITLE
android.permission.MANAGE_DOCUMENTS required

### ISSUE THREAD

**Issue by Dan-Pic** [PhilippC/keepass2android#1021]
Observed with version 1.07b-r0
Database is from Dropbox.
Behaviour not observed previously.
When opening the database, I have a message that corresponds to the following line from the logbook:
java.lang.SecurityException: Permission Denial: opening provider com.android.providers.media.MediaDocumentsProvider from ProcessRecord{38e2322 4030:keepass2android.keepass2android/u0a151} (pid=4030, uid=10151) requires android.permission.MANAGE_DOCUMENTS or android.permission.MANAGE_DOCUMENTS

**Comment by PhilippC** [comment]
it looks like even though your database is stored on Dropbox, you are not using the built-in Dropbox integration but instead opened the file from the Dropbox app. Android revoked the permission for KP2A to access the file originally created/exported from the Dropbox app. You can either re-open the file the way you did it before, but I strongly recommend you choose "Change database", "Open file", "Dropbox" and browse to the database.

**Comment by Dan-Pic** [comment]
Hi,
I never opened the file outside KP2A.
I tried to re-open like you suggested, with the same error. But I noticed
that KP2A remembers the database (it remembers the password method and the
key file).
So finally I deleted the database with KP2A app. Then I opened it (using
the built-in integration). And I have no more error!
I previously used a key file locally exported from Google drive on my
smartphone, while now since I newly  opened the database, I have also used
the built-in integration for the key file. However, I don't think that it
could explain the previous error.
I think that the error happened after KP2A update.
Best regards
